apps/web/src/pages/About/About.tsx

```diff
@@ -36,13 +36,13 @@ const About = ({ members }: AboutProps) => {
     <div className='flex flex-col items-center justify-between gap-16 bg-black px-28 pt-10 pb-20 max-lg:px-[30px]'>
       <p className='text-center text-5xl font-semibold'>{t('title')}</p>
       <div>
-        <div className='flex flex-wrap justify-between gap-10 after:flex-auto after:content-[""] max-lg:justify-center'>
+        <div className='m-x-auto flex max-w-[1100px] flex-row flex-wrap  justify-center gap-2  '>
           {members.map((member) => (
             <Member key={member?.id} member={member} />
           ))}
         </div>
       </div>
-      <NextSeo title={t('seo-title')} />
+      <NextSeo title={t('about')} />
```

Comment:
could please add a description and logo? 🙏 
you can copy it from the home page

Reply:
done